src/App.js

```diff
@@ -14,6 +14,7 @@ function App() {
       <Routes>
```

Comment:
- [ ] This <Routes> should be <Router> kindly fix this, it should be working from the last milestone before approved